packages/lesswrong/server/repos/helpers.ts

```diff
@@ -15,3 +22,11 @@ export const getViewablePostsSelector = (postsTableAlias?: string) => {
     ${aliasPrefix}"postedAt" IS NOT NULL
   `;
 };
+
+export const getViewableTagsSelector = (tagsTableAlias?: string) => {
+  const aliasPrefix = tagsTableAlias ? `${tagsTableAlias}.` : "";
+  return `
+    ${aliasPrefix}"deleted" = FALSE
+    ${aliasPrefix}"adminOnly" = FALSE
```

Comment:
I'm a bit concerned that, especially for Tags, we'll add a condition that makes it not visible and not update this. This is now adding a second source of truth.

Reply:
We have the same issue (preexisting) on posts. I'll add comments on both ends saying to keep this in syn with the default view.